src/views/CollectionIndex/Samples/Page.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import { PageHeader } from "../../../components/Header/PageHeader";
+
+export const Page = (props) => {
+  const { data } = props
+  const [images, selectedImages] = useState([])
+  const [cartCount, addItemIntoCart] = useState(0)
+  const collect = [];
+  localStorage.setItem('show', "true")
+
+  const handleClick = () => {
+    props.history.goBack()
+  }
+
+  const selectImage = (e) => {
+    if(images.indexOf(e.target.id) > -1){
+      const image = images.filter(img => img !== e.target.id)
+      selectedImages(image);
+    }else{
+      selectedImages([...images, e.target.id]);
+    }
+  };
+
+  useEffect(() => {
+    collect[props.collectionId] = [...images]
+  }, [images])
+
+  const handleOrderSamples = () => {
+    const count = collect[props.collectionId].length
+    addItemIntoCart(count)
+    if(count === 0){
+      localStorage.removeItem('cartItems')
+    }else{
+      localStorage.setItem('cartItems', String(count))
+    }
+  }
```

Comment:
This should come from backend. But I guess you're still working on this part?

We probably don't need localStorage for keeping track of cart items.